### PR TITLE
can use condition on sass now

### PR DESCRIPTION
**Pull request by smallst**
this one is simple and aim to merge webpack config so that other pr can use sass condition too

### CODE REVIEW

File: webpack.base.config.js [hunk 1]
```diff
@@ -189,7 +189,8 @@ module.exports = (env) => {
               options: {
                 implementation: require("sass"),
                 sassOptions: {
-                  indentedSyntax: true
+                  indentedSyntax: true,
+                  additionalData: `$product: ${product};`,
```
Comment (adamkecskes) [inline webpack.base.config.js:193]: Why not define `$is-codecombat` already here so we can skip these 
https://github.com/codecombat/codecombat/pull/7014/files#diff-ffdec86f389b986608e7acf6442922c659476fae0afddc7e810204f93d0c8673R388
?

Comment (smallst) [reply webpack.base.config.js:193]: emmm the `additionalData` seems only can pass string in instead of boolean so ... anyway we need a check later 😂

Comment (adamkecskes) [reply webpack.base.config.js:193]: can't you say something like this?
```
additionalData: `$is-codecombat: $product == '${product}';`
```

File: webpack.base.config.js [hunk 2]
```diff
@@ -214,6 +215,7 @@ module.exports = (env) => {
               loader: 'sass-loader',
               options: {
                 implementation: require("sass"),
+                additionalData: `$is-codecombat: ${product == 'codecombat'};`,
```
Comment (adamkecskes) [inline webpack.base.config.js:218]: Don't we need `sassOptions` here as above?

Comment (smallst) [reply webpack.base.config.js:218]: i don't know. but if it works normal, i don't want to change it. since i don't feel so familiar with webpack configs 😂

Comment (mrfinch) [reply webpack.base.config.js:218]: I think we can use it this way: https://webpack.js.org/loaders/sass-loader/#string-1

Also, you can always add a class in HTML based on a variable in Vue template if you don't want to go via this route 😅

Comment (smallst) [reply webpack.base.config.js:218]: oh sorry i don't catch you two yesterday. now it is fixed 😂

**Review by mrfinch — COMMENTED** [review]
testcase failing with this error:
```
Error: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: semicolons aren't allowed in the indented syntax.
  ╷
1 │ $is-codecombat: true;
  │                     ^
  ╵
  ozaria/site/views/play/level/modal/CourseVictoryComponent.vue 1:21  root stylesheet
```

**Comment by smallst** [comment]
> testcase failing with this error:
> 
> ```
> Error: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
> SassError: semicolons aren't allowed in the indented syntax.
>   ╷
> 1 │ $is-codecombat: true;
>   │                     ^
>   ╵
>   ozaria/site/views/play/level/modal/CourseVictoryComponent.vue 1:21  root stylesheet
> ```

good catch! sass doens't use same syntax as scss

**Review by mrfinch — APPROVED** [review]

**Pull request merged**